src/cmd/opt.js

```
@@ -129,19 +156,39 @@
-  const wasmOptPath = fileURLToPath(import.meta.resolve('binaryen/bin/wasm-opt'));
+async function wasmOpt(args) {
+  const {
+    moduleBytes,
+    cliArgs,
+  } = args;
+  // Get wasmOpt binary, ensure it exists
+  const wasmOptPath = env.WASM_OPT_BIN_PATH ?? args?.wasmOptBinPath ?? fileURLToPath(import.meta.resolve('binaryen/bin/wasm-opt'));
+  try {
+    await stat(wasmOptPath);
+  } catch (err)  {
+    if (err && err.code && err.code === 'ENOENT') {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)